sdk/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "kepler-sdk"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+"wasm" = ["kepler-lib/wasm"]
+
+[dependencies]
+base64 = "0.13"
+chrono = { version = "0.4", features = ["wasmbind"] }
```

Comment:
This feature could also be dependent on the `wasm` feature being active.

Reply:
Totally agree (66c9e08)